realm/realm-library/src/main/java/io/realm/OsRealmSchema.java

```diff
@@ -62,6 +64,26 @@ public boolean contains(String className) {
         }
 
         @Override
+        Table getTable(Class<? extends RealmModel> clazz) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        Table getTable(String className) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        RealmObjectSchema getSchemaForClass(Class<? extends RealmModel> clazz) {
+            throw new UnsupportedOperationException();
+        }
+
+        @Override
+        StandardRealmObjectSchema getSchemaForClass(String className) {
```

Comment:
This should be `OsRealmObjectSchema`.
Narrowing return type from parent class is valid since Java5.